_includes/info-slider.scss

```diff
@@ -125,6 +125,11 @@
             background: #{$color-RO_donkerblauw--interaction};
             transition: none;
         }
+
+        &:focus {
+            outline-color: #2A71C3;
+            outline-width: 2px;
```

Comment:
Misschien beter `outline: 2px solid transparent` in de normale variant en hier alleen `outline-color`